WPFHospitalEditor/AllMapObjects.cs

```diff
@@ -12,9 +16,55 @@ class AllMapObjects
         public static List<MapObject> allFirstBuildingSecondFloorObjects = new List<MapObject>();
         public static List<MapObject> allSecondBuildingFirstFloorObjects = new List<MapObject>();
         public static List<MapObject> allSecondBuildingSecondFloorObjects = new List<MapObject>();
+        public static List<MapObject> allMapObjects = new List<MapObject>();
+
+        public static char separator = Path.DirectorySeparatorChar;
+
+        public static string MAPOBJECT_PATH = $"..{separator}..{separator}..{separator}Repository{separator}Data{separator}AllMapObjects.json";
+        FileRepository path1 = new FileRepository(MAPOBJECT_PATH);
+        MapObjectController mapObjectController1 = new MapObjectController(new MapObjectService(new MapObjectRepository(new FileRepository(MAPOBJECT_PATH))));
 
         public AllMapObjects()
+        {                                                 
+            allMapObjects = mapObjectController1.getAllMapObjects();
```

Comment:
Why is there mapObjectController1 when you have mapObjectController?

Reply:
It was a mistake, there was controller for each List in AllMapObject, and when I deleted them I forgot to change controller name.